indico/modules/events/contributions/templates/management/_contribution_report.html

```diff
@@ -1,6 +1,13 @@
 {% from 'message_box.html' import message_box %}
 
-{% macro render_contribution_list(event, contribs, sessions, tracks) %}
+{% macro render_displayed_entries_fragment(displayed_num, total_num) %}
+    <span id="filtering-state" class="i-button label icon-list"
+          title="{% trans %}Displaying {{displayed_num}} out of {{total_num}} contributions{% endtrans %}">
+        <strong>{{ displayed_num }}</strong> / {{ total_num }}
+    </span>
+{% endmacro %}
```

Comment:
The template only contains macros, so the template itself consists of nothing but whitespace when rendered (not that we'd ever do that in the first place) - so it doesn't really matter whether it's stripped or not